src/ploomber_core/warnings.py

```diff
@@ -6,4 +10,26 @@ class PloomberDeprecationWarning(FutureWarning):
         Changed superclass from UserWarning to FutureWarning
     """
 
-    pass
+    # @
+    def __init__(self, message):
+        print("message: ", message)
```

Comment:
no need to print the message

Reply:
I guess if we don't want to post to posthog the comsumer function can just invoke `warn(message, FutureWarning)`